tests/tests.rs

```diff
@@ -1,7 +1,36 @@
-use goldentests::{ TestConfig, TestResult };
+use goldentests::{TestConfig, TestResult};
 
 #[test]
-fn run_goldentests() -> TestResult<()> {
+fn run_goldentests_example() -> TestResult<()> {
     let config = TestConfig::new("python", "examples", "# ")?;
     config.run_tests()
 }
+
+#[test]
+fn test_overwrite() -> TestResult<()> {
+    // Test this without the filesystem? Is it worth it?
```

Comment:
I think it is okay to leave this test out. Once this feature is implemented and verified I don't see it needing continuous testing. I appreciate you staying on top of things though and thinking about this case